pyeffects/Option.py

```diff
@@ -84,6 +84,15 @@ def __init__(self, value: A) -> None:
         self.value = value
         self.biased = True
 
+    def __eq__(self, __value: object) -> bool:
```

Comment:
ditto here.  the variable name, let's change it to `other` and change the return type from `object` to `Option[A]`.